schedule/kernel/agama_gfx_drivers.yaml

```diff
@@ -3,9 +3,9 @@ name: agama-gfx-usb-drivers
 vars:
     AGAMA: 1
```

Comment:
Why we have `AGAMA: 1`  i don't see it in any  active jobs in openQA. This could be dropped.